legal-api/tests/unit/resources/v2/test_business_documents.py

```diff
@@ -76,6 +76,26 @@ def test_get_business_documents(session, client, jwt):
     assert docs_json['documents']
     assert docs_json['documents']['summary']
 
+def test_get_business_documents_ca(requests_mock, session, client, jwt):
+    """Assert that business summary is not returned."""
+    # setup
+    identifier = 'CP7654321'
+    factory_business(identifier)
+    requests_mock.get(
+        f"{current_app.config.get('AUTH_SVC_URL')}/orgs/123456/products?include_hidden=true",
+        json=[{"code": "CA_SEARCH", "subscriptionStatus": "ACTIVE"}],
+    )
+    # test
+    rv = client.get(f'/api/v2/businesses/{identifier}/documents',
+                    headers=create_header(jwt, [STAFF_ROLE], identifier, **{'Account-Id': '123456'})
+                    )
+    # check
+    assert rv.status_code == HTTPStatus.OK
+    docs_json = rv.json
+    assert docs_json['documents']
+    assert docs_json['documents']['summary']
+    assert not docs_json['documents'].get('receipt', False)
+
```

Comment:
Don't believe business level documents have receipts.

You'll probably want to add a test to [test_filing_documents.py](https://github.com/bcgov/lear/blob/main/legal-api/tests/unit/resources/v2/test_business_filings/test_filing_documents.py) to see if receipts are excluded for a given filing when CA is making the call.

Reply:
Added another